src/boilerpipe/extract/__init__.py

```diff
@@ -42,6 +42,16 @@ def __init__(self, extractor='DefaultExtractor', **kwargs):
             if encoding.lower() == 'text/html':
                 encoding = chardet.detect(self.data)['encoding']
             try:
+                import gzip
```

Comment:
nowadays lots and lots of websites are gzip
i added exception "pass" if it's not